### PR TITLE
Fix 1322 downsampling 4 color byte images fully to 3 color byte

### PR DESCRIPTION
**Pull request by therealmarv**
Downsample images with the right ICC profile to 3 color byte images

### CODE REVIEW

File: bakery/src/scripts/Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ LABEL maintainer="OpenStax Content Engineering"
 
 RUN apt-get update
 RUN apt-get install -y build-essential libicu-dev pkg-config libmagic1 mime-support wget curl xsltproc lsb-release git
-RUN apt-get install -y imagemagick icc-profiles-free
+RUN apt-get install -y imagemagick icc-profiles-free curl unzip
```
Comment (rnathuji) [inline bakery/src/scripts/Dockerfile:7]: Minor: I just happened to notice this (and it's not directly related to this PR anyways), but is there some reason why we have two consecutive `apt-get install` steps here in the `Dockerfile`?

File: .circleci/config.yml [hunk 2]
```diff
@@ -75,7 +75,7 @@ jobs:
           name: Setup test environment
           command: |
             sudo apt-get update
-            sudo apt-get install -y imagemagick icc-profiles-free
+            sudo apt-get install -y imagemagick icc-profiles-free ghostscript
```
Comment (rnathuji) [inline .circleci/config.yml:78]: Minor: Should we update this dep in the `README.md` note too?

**Review by rnathuji — APPROVED** [review]

File: bakery/src/scripts/Dockerfile (hunk 1 above)
Comment (therealmarv) [reply bakery/src/scripts/Dockerfile]: I was not aware about that. Just informed myself now about [Docker best practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run) and updated the line.

File: .circleci/config.yml (hunk 2 above)
Comment (therealmarv) [reply .circleci/config.yml:78]: yes that's important for the manual test runs. Updated.

**Review by rnathuji — APPROVED** [review]

**Pull request merged**